CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to Gitpod Samples
+
+Thank you for showing your interest in contributing to Gitpod Samples. We value community contributions 💜 
+
+We celebrate that every person brings a unique perspective and experience to the community. 
+
+Before reporting any issues or pull requests, please read through this document to ensure we have all the information to respond effectively to your contribution.
+
+## Submit a new demo or tutorial using Gitpod 
+
+- Submit a new demo or tutorial ideas by raising a new issue in [Awesome Gitpod](https://github.com/Gitpod-Samples/awesome-gitpod)
+- When reporting an issue, please make sure it hasn't already been reported by someone else by looking through [recently closed](https://github.com/Gitpod-Samples/awesome-gitpod/issues?q=is%3Aissue+is%3Aclosed) or [existing open](https://github.com/Gitpod-Samples/awesome-gitpod/issues?q=is%3Aopen+is%3Aissue) issues.
+
+
+## Submit a new pull request in the existing repository
+
+To submit a Pull request 
+follow our doc on [Submitting a pull request](https://www.gitpod.io/docs/help/contribute/features-and-patches/submitting-a-pull-request)
+
+
+## Code of Conduct 
+
+Please go through our [Code of Conduct](CODE_OF_CONDUCT.md) document
+
+- This Code of Conduct is adapted from the [Contributor Covenant][homepage],
+version 2.0, available at
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+- Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
+
+- For answers to common questions about this code of conduct, see the FAQ at
+https://www.contributor-covenant.org/faq. Translations are available at
+https://www.contributor-covenant.org/translations.
+
+We ❤️ the people involved in Gitpod Samples, and we'd love to help you with contributing. Drop by the `#contributing` channel on the [Gitpod Discord server](https://www.gitpod.io/chat) and _ask for help_.
```

Comment:
I'd reword this part:

We 🧡 all Gitpod Samples contributions! If you have any questions about contributing, drop by the `#contributing` channel on our community Discord server. See you there! 👋🏼